python/moose/fixXreacs.py

```diff
@@ -10,17 +13,16 @@
 # This program carries no warranty whatsoever.
 ####################################################################
 
-
```

Comment:
All instance of `moose` are replaced by `_moose` to reflect that they belong to `_moose.so`. Made changes to `__init__.py` file to make sure API does not change.

Reply:
I am copying the content of email here for the sake of completeness. It is bit edited. @hrani wanted some clarification regarding this change. 

---

In short, these changes only affect developer of python moose module and no one else. GUI and Rdesigneur will see the same API.

This PR makes it explicit weather a function is called from _moose.so (which c++/python module) or moose.py+ other python scripts.  Previously it was not an issue, but recently it has become an issue.

Function `loadModel` was recently implemented in python bypassing _moose.loadModel (c++ function). Since both has the same name, a hack was introduced in https://github.com/BhallaLab/moose-core/blame/8463cc73e5d837ade8e191404359bc278f377b9d/python/moose/moose.py#L65 in which `loadModel` was deleted from globals() so that python function is the
only function left. Clean way to do it: either remove c++ function or call loadModel as _moose.loadModel to make it explicity. 

Other issues are that few functions were introduced recently which are unnecessarily long e.g. moose.mooseReadSBML/moose.mooseWriteSBML instead of good old `moose.readSBML/moose.writeSBML`. I really don't know the reason for this but I am guessing it is to avoid clashing with c++ readSBML function (if there was any).

These changes cleans this up by explicitly forcing the developer to tell explicitly if she is calling the function from c++ _moose.so or pure python function.

WHAT DOES IT MEAN FOR USER:
    Nothing changes from user perspective. All the functions in _moose.so are
    brought in to global namespace in __init__.py file. A test has been added
    just to make sure that API does not break/change :https://github.com/BhallaLab/moose-core/blob/7fad5fbb7a58c39d5560ef5d4a2e930ab4b36a5f/tests/python/test_moose_attribs.py

RDESIGNEUR OR GUI:
    Nothing changes.

Only `python/moose` directory is affected. Developer has to know the function she is calling belong to _moose.so or other python files. This way there will less confusion in future.

Importing is as `_moose` because the c++ module is named `_moose.so` (historically). It can be changed to `moose.so` but then `moose.py` needs to be removed and renamed as  __init__.py`. Other option to call `_moose.so` as `pymoose.so` or `moose_core.so` and import functions from here. Either way the idea behind this PR to make it explicit from where a function is being called.